src/app/dashboard-widgets/applications-widget/applications-pipeline/applications-pipeline.component.less

```diff
@@ -0,0 +1,214 @@
+@import (reference) '../../../../assets/stylesheets/shared/osio.less';
+
```

Comment:
can you swap the hex colors over to PatternFly variables?

Reply:
Done.